### PR TITLE
Add functions to remove, add and edit a task

### PR DESCRIPTION
**Pull request by SFarida**
### Below are the updates on this branch
- Functions to add, delete and edit a task were added in a the newly created file task.js which is imported in the index.js file.
- The tasks are stored in the local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -47,6 +86,23 @@ const renderTasks = () => {
...
+    if (tasks[i].completed === true) {
+      tempTasks = tempTasks.filter((task) => task.index !== tasks[i].index);
+    }
+  }
+  renderTasks();
+});
+
 window.onload = () => {
   renderTasks();
 };
```
Comment (levy002) [inline src/index.js]: - [ ] Kindly make sure to fix the reloading problem of your page which is happening every time we add a new todo.

   **Hint:**    _Remove the `window.onload` event and call only your `renderTasks` function_

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @SFarida   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -103,6 +103,4 @@ completed.addEventListener('click', () => {
   renderTasks();
 });
 
-window.onload = () => {
-  renderTasks();
-};
\ No newline at end of file
+renderTasks();
```
Comment (codepantha) [inline src/index.js]: - [ ] According to the previous reviewer, please ensure that your page doesn't reload when adding a new to-do item. Despite the fact that you have removed the `window.location.reload()` line here, you are still calling the same function in the tasks.js file when adding, editing and deleting tasks.

File: src/modules/tasks.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ addInputField.addEventListener('keypress', (e) => {
     localStorage.setItem('tasks', JSON.stringify(tasks));
     addInputField.value = '';
     window.location.reload();
+    // renderTasks();
```
Comment (codepantha) [inline src/modules/tasks.js]: - [ ] Please remove commented code to make your work less cluttered 👍

File: src/modules/tasks.js [hunk 4]
```diff
@@ -26,6 +27,7 @@ export const removeTask = (taskId) => {
   tasks = tasks.filter((task) => task.index !== taskId);
   fixIndex(tasks);
   window.location.reload();
+  // renderTasks();
```
Comment (codepantha) [inline src/modules/tasks.js]: - [ ] Please remove commented code to make your work less cluttered 👍

File: src/modules/tasks.js [hunk 5]
```diff
@@ -40,6 +42,7 @@ export const editTast = (id) => {
       }
       localStorage.setItem('tasks', JSON.stringify(tasks));
       window.location.reload();
+      // renderTasks();
```
Comment (codepantha) [inline src/modules/tasks.js]: - [ ] Please remove commented code to make your work less cluttered 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @SFarida 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @SFarida, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow
- Requested  changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**